src/phases/faint-phase.ts

```diff
@@ -127,7 +128,7 @@ export class FaintPhase extends PokemonPhase {
 
     pokemon.faintCry(() => {
       if (pokemon instanceof PlayerPokemon) {
-        pokemon.addFriendship(-10);
+        pokemon.addFriendship(FRIENDSHIP_LOSS_FAINT);
```

Comment:
```suggestion
        pokemon.addFriendship(-FRIENDSHIP_LOSS_FAINT);
```
Either the negative should be put on the constant or this should get a `-` attached to it here.

Reply:
lmao so true. Either `pokemon.addFriendship(-1 * FRIENDSHIP_LOSS_ON_FAINT)` or create a new `loseFriendship` method